packages/shared/styles/components/molecules/SfBanner.scss

```diff
@@ -2,23 +2,30 @@
 .sf-banner {
   box-sizing: border-box;
   display: var(--banner-display, flex);
-  align-items: var(--banner-align-items, flex-start);
   justify-content: var(--banner-justify-content, flex-start);
-  min-height: var(--banner-height, 21.4375rem);
   width: var(--banner-width, 100%);
-  padding: var(--banner-padding, var(--spacer-xl));
+  min-height: var(--banner-height, 21.4375rem);
   @include background(
     --banner-background,
     var(--_banner-background-color, transparent),
     var(--_banner-background-image)
   );
   --banner-background-position: 60%;
-  &__container {
+  &__wrapper {
     display: flex;
-    flex: 0 0 var(--banner-container-width, 70%);
+    flex-direction: column;
+    align-items: var(--banner-align-items, flex-start);
     justify-content: var(--banner-container-justify-content, flex-start);
```

Comment:
would be good to change also custom properties names inside wrapper: ```--banner-container-justify-content``` to ```--banner-wrapper-justify-content``` :)

Reply:
changed